modules/crm_core_default_matching_engine/crm_core_default_matching_engine.module

```diff
@@ -158,27 +158,12 @@ function crm_core_default_matching_engine_load_field_config($contact_type, $fiel
 }
 
 /**
- * Implements hook_entity_insert().
- *
- * Create a matching rule for each created contact type.
- */
-function crm_core_default_matching_engine_entity_insert(Drupal\Core\Entity\EntityInterface $entity) {
-  if ($entity->getEntityTypeId() == 'crm_core_contact_type') {
-    MatchingRule::create(array(
-      'type' => $entity->id(),
-      'status' => FALSE,
-    ))
-    ->save();
-  }
-}
-
-/**
  * Implements hook_entity_delete().
  *
  * Remove matching rule for deleted contact type.
  */
 function crm_core_default_matching_engine_entity_delete(Drupal\Core\Entity\EntityInterface $entity) {
   if ($entity->getEntityTypeId() == 'crm_core_contact_type') {
-    MatchingRule::load($entity->id())->delete();
+    Matcher::load($entity->id())->delete();
```

Comment:
This is dropped with crm_core_default_matching_engine module, probably needs some work when we add configuration dependencies.